courses/machine_learning/deepdive2/structured/solutions/3b_bqml_linear_transform_babyweight.ipynb

```diff
@@ -0,0 +1,1162 @@
+{
```

Comment:
<ul><li>Add a markdown cell before this cell with the following comment: "Let's evaluate our model on the eval dataset and store the table back into BiqQuery."</li><li>Use the --destination_table argument of the %%bigquery magic to save back the evaluation table into BQ</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/669/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>Done</p>